components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+                )}
+            </div>
+        </div>
+
+        {createTeamModal && (
+            <NewTeamModal onClose={() => setCreateTeamModal(undefined)} onBuy={onBuy} {...createTeamModal} />
+        )}
+
+        {manageTeamModal && (
+            <ManageTeamModal onClose={() => setManageTeamModal(undefined)} slotInputHandler={slotInputHandler} slots={getSlotsForSub(manageTeamModal.sub)} />
+        )}
+
+        {inviteMembersModal && (
+            <InviteMembersModal onClose={() => setInviteMembersModal(undefined)} {...inviteMembersModal} />
+        )}
+
+        {addMembersModal && (
+            <AddMembersModal onClose={() => setAddMembersModal(undefined)} onBuy={onBuy} {...addMembersModal} />
+        )}
+
+        {(getActiveSubs().length === 0 && !pendingPurchase) && (
+            <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100">
+                <div className="m-auto text-center">
+                    <h3 className="self-center text-gray-500 mb-4">No Active Teams</h3>
+                    <div className="text-gray-500 mb-6">Get started by creating a team<br /> and adding team members. <a href="https://www.gitpod.io/docs/teams" target="_blank" rel="noopener" className="underline underline-thickness-thin underline-offset-small hover:text-gray-600">Learn more</a></div>
+                    <button className="self-center" onClick={() => showCreateTeamModal()}>Create Team</button>
+                </div>
+            </div>
+        )}
+
+        {(getActiveSubs().length > 0 || !!pendingPurchase) && (
+            <div className="flex flex-col pt-6 space-y-2">
+                {pendingPurchase && (
+                    <div key={"team-sub-" + pendingPurchase.plan.chargebeeId} className="flex-grow flex flex-row hover:bg-gray-100 rounded-xl h-16 w-full">
+                        <div className="px-4 self-center w-1/12">
+                            <div className={"rounded-full w-3 h-3 text-sm align-middle bg-gitpod-kumquat"}>
+                                &nbsp;
+                        </div>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto font-medium truncate overflow-ellipsis">{pendingPurchase.plan.name}</span>
+                            <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">Purchased on {formatDate(new Date().toString())}</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{"–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Payment in Progress</span>
+                        </div>
+                        <div className="my-auto flex w-1/12 pl-8">
```

Comment:
**nitpick**: Let's hide the more actions button on the default state as we do with other lists like Integrations, Workspaces, etc.

```suggestion
                        <div className="my-auto flex w-1/12 pl-8 opacity-0 group-hover:opacity-100">
```